tests/common.py

```diff
@@ -21,14 +21,19 @@
 def _reset_schema():
     rest.db.engine.execute('drop schema if exists public cascade;')
     rest.db.engine.execute('drop schema if exists disclosure cascade;')
+    rest.db.engine.execute('drop schema if exists staging cascade;')
     rest.db.engine.execute('create schema public;')
     rest.db.engine.execute('create schema disclosure;')
+    rest.db.engine.execute('create schema staging;')
```

Comment:
Oh, right, I forgot about that!  Yes, that would probably be good to file a separate issue for this, even if only to document the behavior.  Thanks for explaining again, I appreciate it!